presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/PrestoClusterTest.groovy

```diff
@@ -124,6 +131,40 @@ class PrestoClusterTest
   }
 
   @Test
+  void 'single node presto app - adding plugin'()
+  {
+    PrestoCluster prestoCluster = new PrestoCluster(slider, hdfsClient, 'resources-singlenode.json', APP_CONFIG_WITHOUT_CATALOG_TEMPLATE)
+    prestoCluster.withPrestoCluster {
+      prestoCluster.assertThatPrestoIsUpAndRunning(0)
+
+      assertThatAllProcessesAreRunning(prestoCluster)
+
+      assertThatPluginJarExists(prestoCluster)
+
+      assertThatApplicationIsStoppable(prestoCluster)
+    }
+  }
+
+  def assertThatPluginJarExists(PrestoCluster prestoCluster) {
```

Comment:
I don't like this, this is too implementation dependant. It would be nice to run some query in presto which would not work without this jar. For example you can add there a jar which defines a connector and just check if show tables works from that connector. 

There is https://github.com/qubole/presto-kinesis you can use for that. It does not have to be fully configured, it just needs to return list of tables. Or you can use another 3rd party connector or write your own one (basing on blackhole),